### PR TITLE
Fix: Fixed an issue where clicking an empty space would scroll to the top of the file list

### LINKED ISSUE #12523: Bug: Single click on empty space to scroll up
### Description

When I single-click on empty space in the details layout, the page always scrolls up to the last selected item.

### Steps To Reproduce

1. Open the folder where scrolling occurs in the details layout.
2. Scroll down.
3. Single click on empty space.
4. The page scrolls up to the last selected item.

### Requirements

Ensure scrolling does not occur when single-clicking.

### Files Version

2.4.71.0

### Windows Version

Windows 11 22H2

### Log file

n/a

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #12523 

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open a folder using `DetailsLayout`
   2. Scroll down
   3. Click on an empty space

### CODE REVIEW

File: src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs [hunk 1]
```diff
@@ -876,5 +876,10 @@ private void SetToolTip(TextBlock textBlock)
 		{
 			ToolTipService.SetToolTip(textBlock, textBlock.IsTextTrimmed ? textBlock.Text : null);
 		}
+
+		private void FileList_LosingFocus(UIElement sender, LosingFocusEventArgs args)
+		{
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs:881]:
```suggestion
		{
		// Fixes an issue where clicking an empty space would scroll to the top of the file list 
```

Comment (ferrariofilippo) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs:881]: I pushed a commit to indent the comment

**Review by yaira2 — APPROVED** [review]

**Comment by hishitetsu** [comment]
@yaira2 @ferrariofilippo this must be reverted. This makes it impossible to type in the path bar or search box.

**Comment by ferrariofilippo** [comment]
I have a fix

**Pull request merged**